d2l-button-subtle.html

```diff
@@ -155,15 +149,47 @@
 				text: {
 					type: String,
 					reflectToAttribute: true
+				},
+
+				/**
+				 * Horizontal alignment of button. Options:
+				 * 	"text" -  The button icon or text will left align with the page content
+				 *	default - The button's edge (including padding) will left align with the page content
+				*/
+				hAlign: {
+					type: String,
+					reflectToAttribute: true
+				},
+
+				isRtl: {
+					type: Boolean,
+					value: false,
+					computed: '_calculateRTL(isAttached)',
+					reflectToAttribute: true
 				}
 
 			},
 
 			behaviors: [
 				D2L.PolymerBehaviors.Button.Behavior,
 				D2L.PolymerBehaviors.FocusableBehavior
-			]
+			],
 
+			_calculateRTL: function(isAttached) {
```

Comment:
This looks repeated.  ;)  A couple different approaches if it turns out we need a calculate RTL method...

1. move to button behavior as a helper method;

2. move to a directionality helper in [polymer-behaviors](https://github.com/Brightspace/d2l-polymer-behaviors-ui), wrap in fast-dom if appropriate, and cache the result since we never really expect it to change after initial page load of our application.